Feature/alpha

PattaFeuFeu opened the pull request:
- [X] Adds optional alpha value slider
- [X] Adds custom SeekBar view that shows the progress as a label text floating above
- [X] Deprecates OnColorSelected callback, adds new, renamed callback interface
- [X] Adds new screenshots including alpha value sliders
- [X] Updates Readme.md accordingly

Pes8 commented:
Thank you @PattaFeuFeu for all the improvements :) Only one thing: I'm not sure it's a good idea to broke the compatibilty with the old callback :/ Maybe we can set it as deprecated but also call it ?  Anyway I'll merge it and tomorrow I'll upload it to bintray :)

PattaFeuFeu commented:
I wasn’t sure either but in the end decided that it’d be OK—I marked the interface as well as the method as deprecated, changed the version number from 1.0.* to 1.1.* so that everyone who currently has gradle set to download anything after 1.0.* wouldn’t be affected by those breaking changes, and added the paragraph in the Readme, although to be fair I guess nobody will read that. 😆

Pes8 commented:
Yeah, that's the point :D Nobody will read the Readme :D Well, I think its OK. I'll upload the new version.

PS: thank you very much, next time add your credits to the readme (if you want). Maybe we can make a "contributors" section
